File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string merge;
        int i = 0, j = 0;
        while(i < word1.size() || j < word2.size())
        {
            if(i < word1.size()) merge.push_back(word1[i++]);
            if(j < word2.size()) merge.push_back(word2[j++]);
        }      
        return merge;
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.js
/**
 * @param {string} word1
 * @param {string} word2
 * @return {string}
 */
var mergeAlternately = function(word1, word2) {
    var merge  = '';
    let i = 0;
    let j = 0;
    while(i < word1.length || j < word2.length)
    {
        if (i < word1.length) merge += word1[i++];
        if (j < word2.length) merge += word2[j++];
    }
    return merge
};